<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManagerReserveWidget</class>
 <widget class="QWidget" name="ManagerReserveWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1280</width>
    <height>165</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QSplitter" name="splitter">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>803</width>
     <height>161</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <widget class="BaseTableView" name="table_widget">
    <property name="editTriggers">
     <set>QAbstractItemView::NoEditTriggers</set>
    </property>
    <property name="selectionMode">
     <enum>QAbstractItemView::SingleSelection</enum>
    </property>
    <property name="selectionBehavior">
     <enum>QAbstractItemView::SelectRows</enum>
    </property>
   </widget>
   <widget class="QFrame" name="right_splitter_frame">
    <layout class="QHBoxLayout" name="right_layout">
     <property name="spacing">
      <number>1</number>
     </property>
     <item>
      <widget class="QListView" name="docs_list">
       <property name="selectionMode">
        <enum>QAbstractItemView::ContiguousSelection</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QSplitter" name="smaller_splitter">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <widget class="QFrame" name="right_frame">
        <property name="frameShape">
         <enum>QFrame::Box</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="sum_frame">
           <property name="frameShape">
            <enum>QFrame::Box</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="sum_layout_2">
            <property name="margin">
             <number>0</number>
            </property>
            <item>
             <widget class="QLabel" name="num_label">
              <property name="font">
               <font>
                <weight>75</weight>
                <bold>true</bold>
               </font>
              </property>
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <property name="frameShadow">
               <enum>QFrame::Sunken</enum>
              </property>
              <property name="text">
               <string>TextLabel</string>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="commonsumm_label">
              <property name="text">
               <string>Общая сумма: </string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="summ_label">
              <property name="text">
               <string>0</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="rub_label">
              <property name="text">
               <string>р.</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QFrame" name="grid_lt">
           <layout class="QGridLayout" name="gridLayout">
            <property name="horizontalSpacing">
             <number>3</number>
            </property>
            <property name="verticalSpacing">
             <number>2</number>
            </property>
            <property name="margin">
             <number>3</number>
            </property>
            <item row="0" column="0">
             <widget class="QLabel" name="to_label">
              <property name="text">
               <string>Контрагент:</string>
              </property>
             </widget>
            </item>
            <item row="0" column="1">
             <widget class="DoubleClickLineEdit" name="note_edit">
              <property name="readOnly">
               <bool>true</bool>
              </property>
             </widget>
            </item>
            <item row="1" column="0">
             <widget class="QLabel" name="who_label">
              <property name="text">
               <string>Для кого:</string>
              </property>
             </widget>
            </item>
            <item row="1" column="1">
             <widget class="QComboBox" name="who_combobox"/>
            </item>
            <item row="0" column="2">
             <widget class="QPushButton" name="clear_note">
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>25</height>
               </size>
              </property>
              <property name="text">
               <string>X</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <spacer name="verticalSpacer">
           <property name="orientation">
            <enum>Qt::Vertical</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>20</width>
             <height>40</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QWidget" name="buttons_widget" native="true">
           <layout class="QHBoxLayout" name="buttons_layout_2">
            <property name="spacing">
             <number>3</number>
            </property>
            <property name="margin">
             <number>3</number>
            </property>
            <item>
             <widget class="QPushButton" name="plus_button">
              <property name="minimumSize">
               <size>
                <width>25</width>
                <height>55</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>25</width>
                <height>55</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Добавить позицию</string>
              </property>
              <property name="text">
               <string>+</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QVBoxLayout" name="destroy_buttons_layout">
              <item>
               <widget class="QPushButton" name="minus_button">
                <property name="minimumSize">
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Вычесть количество</string>
                </property>
                <property name="text">
                 <string>-</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="delete_button">
                <property name="minimumSize">
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>25</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Удалить запись</string>
                </property>
                <property name="text">
                 <string>X</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="move_buttons_layout">
              <item>
               <widget class="QPushButton" name="show_button">
                <property name="minimumSize">
                 <size>
                  <width>75</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>75</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Просмотр заявки</string>
                </property>
                <property name="text">
                 <string>Просмотр</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="clear_button">
                <property name="minimumSize">
                 <size>
                  <width>75</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>75</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Вернуть всё в базу</string>
                </property>
                <property name="text">
                 <string>Очистить</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QVBoxLayout" name="move_layout">
              <item>
               <widget class="QPushButton" name="move_button">
                <property name="minimumSize">
                 <size>
                  <width>55</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Переместить в заявку или в документ</string>
                </property>
                <property name="text">
                 <string>Переместить</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="close_button">
                <property name="minimumSize">
                 <size>
                  <width>55</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="maximumSize">
                 <size>
                  <width>100</width>
                  <height>25</height>
                 </size>
                </property>
                <property name="toolTip">
                 <string>Закрыть окно</string>
                </property>
                <property name="text">
                 <string>Закрыть</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <widget class="QPushButton" name="tobase_button">
              <property name="minimumSize">
               <size>
                <width>55</width>
                <height>55</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>55</width>
                <height>55</height>
               </size>
              </property>
              <property name="toolTip">
               <string>Создать документ</string>
              </property>
              <property name="text">
               <string>В базу</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="fake_label">
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
     </item>
     <item>
      <spacer name="right_spacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>9</width>
         <height>156</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>DoubleClickLineEdit</class>
   <extends>QLineEdit</extends>
   <header location="global">common/helpers.h</header>
  </customwidget>
  <customwidget>
   <class>BaseTableView</class>
   <extends>QTableView</extends>
   <header location="global">common/basetableview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
